package-lock.json

```diff
@@ -1,801 +1,8 @@
 {
   "name": "users-app-backend",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
It looks like your computer might be on a different version of npm so it altered the format of `package-lock.json`. In general, you should not commit `package-lock.json` unless you have added or removed a package from `package.json` or changed the required version of a package in `package.json`. In other words, if there are no changes to `package.json`, don't commit the changes to `package-lock.json`. This is an annoying problem with `npm` that isn't your fault -- I don't know why npm makes unnecessary changes to the lock file like this.